point_cloud_transport/include/point_cloud_transport/point_cloud_transport.hpp

```diff
@@ -264,25 +264,25 @@ class PointCloudTransport : public PointCloudTransportLoader
     const std::string & base_topic, rmw_qos_profile_t custom_qos,
     void (T::* fp)(const sensor_msgs::msg::PointCloud2::ConstSharedPtr &) const, T * obj,
     const point_cloud_transport::TransportHints * transport_hints = nullptr,
-    bool allow_concurrent_callbacks = false)
+    bool /*allow_concurrent_callbacks*/ = false)
```

Comment:
I'm going to suggest that instead of doing this, we actually just remove this parameter completely.  As far as I can tell, it is never used by any callers or callees.